pkgs/development/python-modules/apricot-select/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, numpy
+, scipy
+, tqdm
+, scikit-learn
+, nose
+, numba
+, torch
+}:
+
+
+buildPythonPackage rec {
+  pname = "apricot-select";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    repo = "apricot";
+    owner = "jmschrei";
+    # no tags for recent versions: https://github.com/jmschrei/pomegranate/issues/974
```

Comment:
Tags exist now. Can we use those instead?

Reply:
Sorry the URL in the comment points to the wrong issue/repository. Pomegranate has tags and the issue is closed.

Reply:
Changed to fetch from pypi instead.